app/helpers/taxons_helper.rb

```diff
@@ -0,0 +1,11 @@
+module TaxonsHelper
```

Comment:
Might be that this is hinting towards content item being a model. That seems a later concern though and removing this presenter class looks a nice improvement.

Reply:
Yeah, I did wonder about that. I think we need some more examples before modelling it, as the current usage is so specific to a particular document type.